internal/operator/cluster/cluster.go

```diff
@@ -59,9 +59,9 @@ const ReplicaSuffix = "-replica"
 
 // contstants defining the names of the various sidecar containers
 const (
-	collectCCPImage    = "crunchy-collect"
-	pgBadgerCCPImage   = "crunchy-pgbadger"
-	crunchyadmCCPImage = "crunchy-admin"
+	postgresexporterPGOImage = "crunchy-postgres-exporter"
```

Comment:
I don't see this constant being used. Can we remove it?

Reply:
Yes, those all three constants are no longer in use, so I removed them.